products/compute/api.yaml

```diff
@@ -935,6 +935,266 @@ objects:
         description: |
           How many seconds to wait for the backend before considering it a
           failed request. Default is 30 seconds. Valid range is [1, 86400].
+  - !ruby/object:Api::Resource
+    name: 'RegionBackendService'
+    kind: 'compute#backendService'
+    base_url: projects/{{project}}/regions/{{region}}/backendServices
+    collection_url_response: !ruby/object:Api::Resource::ResponseList
+      kind: 'compute#backendServicesList'
+      items: 'items'
+    has_self_link: true
+    description: |
+      Creates a BackendService resource in the specified project using the data
+      included in the request.
+    async: !ruby/object:Api::Async
+      operation: !ruby/object:Api::Async::Operation
+        kind: 'compute#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/regions/{{region}}/operations/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::Async::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::Async::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::Async::Error
+        path: 'error/errors'
+        message: 'message'
+    properties:
+      - !ruby/object:Api::Type::Array
+        name: 'backends'
+        description: |
+          The list of backends that serve this BackendService.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'description'
+              description: |
+                An optional description of this resource.
+                Provide this property when you create the resource.
+            - !ruby/object:Api::Type::String
+              name: 'group'
+              description: |
+                The fully-qualified URL of an Instance Group. This defines the list
+                of instances that serve traffic. Member virtual machine
+                instances from each instance group must live in the same zone as
+                the instance group itself. No two backends in a backend service
+                are allowed to use same Instance Group resource.
+
+                Note that you must specify an Instance Group
+                resource using the fully-qualified URL, rather than a
+                partial URL.
+
+                When the BackendService has load balancing scheme INTERNAL, the
+                instance group must be within the same region as the
+                BackendService.
+      - !ruby/object:Api::Type::Enum
+        name: 'loadBalancingScheme'
```

Comment:
This needs to be included in the API call, but must always be `INTERNAL`. Is there a better way to handle this?

Reply:
Omitting it entirely or using an encoder. This is fine though.